### DISCUSSION TITLE
Welcome to phoenix Discussions!

### DISCUSSION THREAD

**Discussion by mikeldking** [Arize-ai/phoenix#395] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by axiomofjoy** [comment]
👋 Hey all!
😄 My name is Xander. 
🔥 🐦 I'm a contributor on Phoenix.
🌍 I'll be active in community, so feel free to reach out!
🛫 Looking forward to helping this project take off!
